client/components/Entry/EntryPage.vue

```diff
@@ -45,7 +46,7 @@ export default {
             title: `${this.$store.state.account.name}'s Health Journal`,
             enableBack: true,
             headerLinks: {
-                "/logs/all": "All",
+                "/logs/all": "All", // keeping the /all here to avoid weird activeLink styling behavior
```

Comment:
I don't think the comment is necessary. This is the intended way to use the header.

Reply:
I see. I'll merge for now to avoid needing another approval, but will remove this comment next PR.